local-dev/sqldatabases.bicep

```diff
@@ -75,7 +65,7 @@ resource sql 'apps/Deployment@v1' = {
           {
             // This container is the running sql instance.
             name: 'sql'
-            image: 'mcr.microsoft.com/azure-sql-edge:${tag}'
+            image: 'mcr.microsoft.com/mssql/server:2022-latest'
```

Comment:
Why are we changing the container image as part of this PR? Did azure sql edge not work as expected?